Hello.txt

```diff
@@ -0,0 +1 @@
+Hello
```

Comment:
I think you should add a period here.

Reply:
Not gonna happen

Reply:
fine, I'll approve. but I don't like it.